### ISSUE TITLE
DOI export request 

### ISSUE THREAD

**Issue by williamsongcn** [scholarly-python-package/scholarly#499]
Some software like zotero require the DOI link to download the pdf files, could you please write a DOI export function

**Comment by arunkannawadi** [comment]
Is that feature available on Google Scholar? If not, I'm afraid we won't be able to implement it either.

**Comment by williamsongcn** [comment]
no, but, there is a 'pub_url' in the return of "rearch_query", could you please add it to  scholarly.bibtex,
for example,
@article{wu2023halide,
 abstract = {of high-performance X-ray detectors still facing some  progress of different types of perovskite  X-ray detectors in recent years,  X-ray detection modes, including direct and indirect X-ray},
 author = {Wu, Ya and Feng, Jiangshan and Yang, Zhou and Liu, Yucheng and Liu, Shengzhong},
 journal = {Advanced Science},
 number = {1},
 pages = {2205536},
 pub_year = {2023},
 publisher = {Wiley Online Library},
 title = {Halide Perovskite: A Promising Candidate for Next-Generation X-Ray Detectors},
 venue = {Advanced Science},
 volume = {10},
 URL={https://onlinelibrary.wiley.com/doi/abs/10.1002/advs.202205536}
}
pub_url will help softwares like zotero download pdf as well. that will help me a lot.
Thank you
